server/controllers/apply/applications/checkYourAnswersController.ts

```diff
@@ -0,0 +1,21 @@
+import type { Request, Response, RequestHandler, NextFunction } from 'express'
+
+import { ApplicationService } from '../../../services'
+import { sections } from '../../../form-pages/apply'
+
+export default class CheckYourAnswers {
+  constructor(private readonly applicationService: ApplicationService) {}
+
+  show(): RequestHandler {
+    return async (req: Request, res: Response, next: NextFunction) => {
+      const application = await this.applicationService.findApplication(req.user.token, req.params.id)
+      const responses = this.applicationService.getResponsesAsSummaryListItems(application)
+
+      res.render(`applications/check-your-answers`, {
+        sections,
+        application,
+        responses,
```

Comment:
In most controllers we pass the heading through here, would be good to follow the pattern here